File: documentation-examples-java/src/java/dev/typr/foundations/docs/db2/BinaryTypes.java
package dev.typr.foundations.docs.db2;

import dev.typr.foundations.Db2Type;
import dev.typr.foundations.Db2Types;

@SuppressWarnings("unused")
public class BinaryTypes {
    //start
    Db2Type<byte[]> binaryType = Db2Types.binary;
    Db2Type<byte[]> binary16 = Db2Types.binary(16);
    Db2Type<byte[]> varbinaryType = Db2Types.varbinary;
    Db2Type<byte[]> blobType = Db2Types.blob;
    //stop
}

File: documentation-examples-scala/src/scala/dev/typr/foundations/docs/db2/BinaryTypes.scala
package dev.typr.foundations.docs.db2
import dev.typr.scalafoundations.*
import dev.typr.scalafoundations.data.*



@SuppressWarnings(Array("unused"))
object BinaryTypes:
  //start
  val binaryType: Db2Type[Array[Byte]] = Db2Types.binary
  val binary16: Db2Type[Array[Byte]] = Db2Types.binary(16)
  val varbinaryType: Db2Type[Array[Byte]] = Db2Types.varbinary
  val blobType: Db2Type[Array[Byte]] = Db2Types.blob
  //stop
